mentor-mind-app/src/pages/VideoPage.tsx

```diff
@@ -1,12 +1,20 @@
 import { Box, Grid, Button } from "@mui/material";
 import Comments from "../components/comments/comments";
 import PinnedContainer from "../components/containers/PinnedContainer";
+import RenderVideoComponent from "../components/render/RenderVideoComponent";
 
 const VideoPage = () => {
   return (
     <Grid container spacing={2} sx={{ display: "flex", flexDirection: "row" }}>
       <Grid sx={{ height: "700px", width: "1000px", marginLeft: "40px", marginTop: "50px", backgroundColor: "var(--primary-color)" }}>
```

Comment:
I think on the design the background color was white (`--secondary-color`) with boxShadow